code/controllers/configuration/entries/game_options.dm

```diff
@@ -107,9 +107,10 @@
 
 /datum/config_entry/flag/protect_assistant_from_antagonist //If assistants can be traitor/cult/other
 
-/datum/config_entry/flag/enforce_human_authority //If non-human species are barred from joining as a head of staff
-
-/datum/config_entry/flag/enforce_human_authority_on_everyone //If non-human species are barred from joining as a head of staff, including jobs flagged as allowed for non-humans, ie. Quartermaster.
```

Comment:
Please use the depreciated config system to ensure old configs are read and converted to 2 or 3.